src/vs/workbench/contrib/chat/browser/codeBlockPart.css

```diff
@@ -64,8 +64,15 @@
 }
 
 .interactive-result-code-block {
+	box-shadow: 0px 2px 16px 0px var(--colors-Effects-Shadow, rgba(0, 0, 0, 0.10));
+}
+
+.interactive-session .interactive-request .interactive-result-code-block {
+	border: 1px solid var(--vscode-editor-wordHighlightStrongBackground);
```

Comment:
We have an existing token for this, we should use it. Unless existing themes will be totally incompatible with the new design and we need to change the name